Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2107,6 +2292,22 @@ def init_globals(command_name: str = ''):
     PAGINATED_COUNT = int(demisto.params().get('paginated_count', '200'))
     ENABLE_DM = demisto.params().get('enable_dm', True)
     PERMITTED_NOTIFICATION_TYPES = demisto.params().get('permitted_notifications', [])
+    common_channels = demisto.params().get('common_channels', None)
+    if common_channels:
+        COMMON_CHANNELS = dict(item.split(':') for item in common_channels.split(','))
+    else:
+        COMMON_CHANNELS = {}
+    SAFE_MODE = demisto.params().get('safe_mode', True)
+
+    # Formats the error message for the 'Channel Not Found' errors
+    error_str = 'The channel was not found'
+    if SAFE_MODE:
+        error_str += ' and Safe Mode is enabled. If this command worked previously for you, please try disabling Safe ' \
+                     'Mode from the instance configuration.'
+    else:
+        error_str += ' Either the Slack app is not a member of the channel, or the slack app does not have permission' \
+                     ' to find the channel.'
+    CHANNEL_NOT_FOUND_ERROR_MSG = error_str
```

Comment:
@yuvalbenshalom I removed the repeating lines for the error message and now just format it based on the setting of the safe mode param.